packages/strapi-plugin-email/services/Email.js

```diff
@@ -2,8 +2,13 @@
 
 const _ = require('lodash');
 
-const getProviderConfig = () => {
-  return strapi.plugins.email.config;
+const getProviderSettings = () => {
```

Comment:
I don't think that it is necessary since you can't change the provider why not just show then one enabled  only. Would avoid having to check the dependencies :D

Reply:
@alexandrebodin @MattieBelt what was the reason for renaming this function? I was just wondering since it is a breaking change. Luckily, I noticed since all our email integration tests failed ;-) Maybe it would have been good to quickly mention it in the release notes? No offense, I truely appreciate your work on Strapi.